gflownet/envs/base.py

```diff
@@ -106,7 +106,7 @@ def proxy2reward(self, proxy_vals):
             proxy_vals = proxy_vals * self.energies_stats[3] + self.energies_stats[2]
         if self.reward_func == "power":
             return np.clip(
-                (-1.0 * proxy_vals / self.reward_norm) ** self.reward_beta,
+                (proxy_vals / self.reward_norm) ** self.reward_beta,
```

Comment:
(to discuss): make code compatible with positive energies,
 `assert self.reward_norm > 0` because `np.log(self.reward_norm)` cannot take negative input

Reply:
We need to discuss this indeed. The convention in the repository is that oracles return negative values (energies), then the -1 will make the argument of the rewards positive.